pkg/northbound/gnmi/set.go

```diff
@@ -73,12 +73,16 @@ func (s *Server) Set(ctx context.Context, req *gnmi.SetRequest) (*gnmi.SetRespon
 		return nil, err
 	}
 	updateResults := make([]*gnmi.UpdateResult, 0)
-	configName := "Running"
```

Comment:
we ought to come up with some constants for the gNMI Nb generated configurations since gNMI can't specify nor this nor the User

Reply:
I'd suggest <targetName>Running